cwms-data-api/src/test/resources/cwms/cda/data/dto/watersupply/waterusercontract.json

```diff
@@ -0,0 +1,123 @@
+{
+  "water-user-contract-ref": {
```

Comment:
Just thinking semantically here, the "reference" has more data than the "contract"
I suspect the content is correct but the naming itself is off slightly.

Also is this contract from the contract perspective or the water users' perspective?
Naming wise looks like user perspective.

I'm not vested in either way, but it's not super clear on initial read.

Reply:
Changed naming scheme in recent commit

Reply:
Renamed to water-user. This contract should be from the contract perspective